### DISCUSSION TITLE
vercel does not deploy commits on github's team project

### DISCUSSION THREAD

**Discussion by m0x61h0x64i** [vercel/vercel#8797] [Help] [answered]
hello, i already connected my github account to the vercel team, the team is pro : 

![image](https://user-images.githubusercontent.com/109550653/198712690-6b953ed9-5618-4163-9b15-b65751431884.png)

i commited new changes and waited for vercel to deploy it but it doesn't work!

and also disconnected my github account from account and connected it to the team account.

and also the above account (which has a blue picture) is a member of team account (pro account which has orange picture).

**Comment by amyegan** [comment, marked as answer]
Hi @m0x31h0x64i! I see that you connected your GitHub account to the team, but did you connect the repo to your Project in the Project Settings? It would look something like this if the Project is already connected to a git repo:

<img width="1004" alt="image" src="https://user-images.githubusercontent.com/8247105/198730102-e287ce0b-b8b9-4ee5-bfca-0db5ab0886ba.png">

Otherwise, you will be presented with options for connecting to a git repository from GitHub, GitLab, or Bitbucket:

<img width="953" alt="image" src="https://user-images.githubusercontent.com/8247105/198730388-018ae229-c662-49b2-bb56-32c72495128f.png">

**Reply by m0x61h0x64i** [reply]
thanks for your reply, yes i already connected github repo to the project on project settings - git tab and commited some test changes several times but that didnt work! i cant deploy my project on the pro vercel using github.

**Reply by m0x61h0x64i** [reply]
> Hi @m0x31h0x64i! I see that you connected your GitHub account to the team, but did you connect the repo to your Project in the Project Settings? It would look something like this if the Project is already connected to a git repo:
> 
> <img alt="image" width="1004" src="https://user-images.githubusercontent.com/8247105/198730102-e287ce0b-b8b9-4ee5-bfca-0db5ab0886ba.png">
> 
> Otherwise, you will be presented with options for connecting to a git repository from GitHub, GitLab, or Bitbucket:
> 
> <img alt="image" width="953" src="https://user-images.githubusercontent.com/8247105/198730388-018ae229-c662-49b2-bb56-32c72495128f.png">

i removed the project and tried to add the project again it deployed my project but it doesn't deploy new incoming commits. what should i do ?

**Reply by amyegan** [reply]
We'll need to dig into more account details to figure out what's going wrong. Do you mind sharing the name of the Project so I can take a look at the settings and deployment logs? 

If you prefer to not to publicly share the project name, I totally understand! In that case it would be best to [submit a case](https://vercel.com/help#issues) to the Customer Success Team who can work with you privately to solve this problem.
